GoogleForCreators/web-stories-wp
# Dashboard: Add new arrow svg & give Icons storybook

## Summary
Adding new arrow SVG to be used in detail template pagination and 'scroll up' button 

## Relevant Technical Choices
- Swapping out SVGs for the above
- moved width/height for template detail pagination svgs into the detail.js file since the size is local to the component and this is what our pattern has ended up being.
- Adding storybook to icons so we can see what we have, gave everything consistent naming and imports while at it. 
- Left names inside of components the same to minimize code change 
- disabled namespace eslint rule in icons stories so that i can loop through each icon without having to manually add it. 


## User-facing changes
- The arrows on template details now look like `<` and `>` 
- The scroll up arrow now looks like `^` 
![new arrow](https://user-images.githubusercontent.com/10720454/82612990-20b06280-9b79-11ea-95f6-089917b3aa7b.gif)

<img width="146" alt="Screen Shot 2020-05-21 at 3 30 21 PM" src="https://user-images.githubusercontent.com/10720454/82612992-21e18f80-9b79-11ea-80bf-b5d8f7345575.png">

## Testing Instructions
- Verify that the icons for the above are indeed updated. (new design video for ref: https://drive.google.com/drive/folders/1onGbNRFVAalJXtbYYFigSbdG-nHUE7As)
- Since I gave the icons an index and am importing from the index now if anything is broken it'd be extremely obvious with the import breaking. I believe I got everything updated but clicking around storybook and dashboard to to a regression test would be nice. 

## Todo
- Consistent naming of icons once we have continuity. I named the new arrows pretty specifically just to keep them in their own space but suspect that'll be temporary
---



Fixes #1514 #1321
